irods-vfs-impl/src/main/java/org/irods/nfsrods/vfs/IRODSVirtualFileSystem.java

```diff
@@ -215,14 +215,18 @@ public int access(Inode _inode, int _mode) throws IOException
 
         Path path = getPath(toInodeNumber(_inode));
 
-        log_.debug("access - _inode path  = {}", path);
-        log_.debug("access - _mode        = {}", _mode);
-        log_.debug("access - _mode & ACCESS4_READ    = {}", (_mode & ACCESS4_READ));
-        log_.debug("access - _mode & ACCESS4_LOOKUP  = {}", (_mode & ACCESS4_LOOKUP));
-        log_.debug("access - _mode & ACCESS4_MODIFY  = {}", (_mode & ACCESS4_MODIFY));
-        log_.debug("access - _mode & ACCESS4_EXTEND  = {}", (_mode & ACCESS4_EXTEND));
-        log_.debug("access - _mode & ACCESS4_DELETE  = {}", (_mode & ACCESS4_DELETE));
-        log_.debug("access - _mode & ACCESS4_EXECUTE = {}", (_mode & ACCESS4_EXECUTE));
+        log_.debug("""
+        		   access - _inode path             = {}
```

Comment:
whitespace?

Reply:
vim vs eclipse formatting. i'll fix these before we merge this PR.